bindata/network/ovn-kubernetes/ovnkube-relay.yaml

```diff
@@ -0,0 +1,165 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: ovnkube-relay
+  namespace: openshift-ovn-kubernetes
+  annotations:
+    kubernetes.io/description: |
+      This daemonset launches the ovn-kubernetes OVN relay networking components.
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: ovnkube-relay
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 10%
+  template:
+    metadata:
+      annotations:
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+      labels:
+        app: ovnkube-relay
+        component: network
+        type: infra
+        openshift.io/component: network
+        kubernetes.io/os: "linux"
+    spec:
+      serviceAccountName: ovn-kubernetes-node
+      hostNetwork: true
+      hostPID: true
+      priorityClassName: "system-node-critical"
+      # volumes in all containers:
+      # (container) -> (host)
+      # /etc/openvswitch -> /etc/openvswitch - ovsdb system id
+      # /var/lib/openvswitch -> /var/lib/openvswitch/data - ovsdb data
+      # /run/openvswitch -> tmpfs - ovsdb sockets
+      # /env -> configmap env-overrides - debug overrides
+      containers:
+      # ovn-relay: programs the vswitch with flows from the sbdb
+      - name: ovn-relay
+        image: "{{.OvnImage}}"
+        #image: registry.build01.ci.openshift.org/ci-ln-q4s05d2/stable@sha256:d800722549b1662175495416177ad1364faa9077c185e5fa00573c4abd7eb81e
+        command:
+        - /bin/bash
+        - -c
+        - |
+          set -e
+          if [[ -f "/env/${K8S_NODE}" ]]; then
+            set -o allexport
+            source "/env/${K8S_NODE}"
+            set +o allexport
+          fi  
+          
+          echo "$(date -Iseconds) - starting ovsdb-server as ovn-relay"
+          ovsdb-server -vconsole:emer -vsyslog:err -vfile:info \
```

Comment:
@mccv1r0 still need --remote=pssl:${K8S_NODE_IP}:{{.OVN_SB_PORT}} so that ovn-controllers can talk to us